Bug-Debian: http://bugs.debian.org/644025
commit 0af966b63fcc55b36380d6538dfb30000f71fef9
Author: Nils Philippsen <nils@redhat.com>
Date:   Wed Feb 2 17:11:28 2011 +0100

    Bug 641259 - [abrt] gimp-2:2.6.11-1.fc14: py-slice.py:172:slice:TypeError: integer argument expected, got float
    
    py-slice: cast cellspacing to int in pyslice() to avoid tracebacks

Index: gimp/plug-ins/pygimp/plug-ins/py-slice.py
===================================================================
--- gimp.orig/plug-ins/pygimp/plug-ins/py-slice.py
+++ gimp/plug-ins/pygimp/plug-ins/py-slice.py
@@ -36,6 +36,9 @@ gettext.install("gimp20-python", gimp.lo
 def pyslice(image, drawable, save_path, html_filename,
             image_basename, image_extension, separate,
             image_path, cellspacing, animate, skip_caps):
+
+    cellspacing = int (cellspacing)
+
     if animate:
         count = 0
         drw = []
